src/Service/ActivityPubManager.php

```diff
@@ -266,6 +283,13 @@ public function updateUser(string $actorUrl): ?User
         $user = $this->userRepository->findOneBy(['apProfileId' => $actorUrl]);
 
         $actor = $this->apHttpClient->getActorObject($actorUrl);
+
+        if (isset($actor['type']) && 'Tombstone' === $actor['type'] && $user instanceof User) {
```

Comment:
Also check here if `$actor` exists/defined. Same for `$user` I guess.

Reply:
User is checked through `instanceof`. The other one will be added